src/components/MainNavbar.js

```diff
@@ -41,7 +41,7 @@ const MainNavbar = () => {
       <ExternalLink
         onClick={() => toggleMenu()}
         key="documentation"
-        href="https://smswithoutborders.github.io/docs/developers/introduction"
+        href="https://smswithoutborders.github.io/docs/contributing/introduction/"
```

Comment:
It will be great to have this link point to [docs for the developer section](https://smswithoutborders.github.io/docs/developers/account/) rather than the contribution docs. The idea here is developers are looking to build with SMSWithoutBorders products like OpenAPI not contribute

Reply:
ok @nSedrickm , i'll update this